### ISSUE TITLE
Cannot read MIRIAD files: missing antpos keyword

### ISSUE THREAD

**Issue by aelanman** [HERA-Team/pyuvdata#126]
I have MIRIAD files generated by pyuvdata that cannot be read by it, because they lack the "antpos" keyword.

Line 59 of pyuvdata/miriad.py returns KeyError: 'antpos'

**Comment by aelanman** [comment]
I wrote this file from FHD save files on Feb 9th.

**Comment by bhazelton** [comment]
Ah, that explains it. Currently FHD does not save or report the antenna positions, so we cannot recover them when we write a Miriad file (they are technically not a required parameter on UVData for this reason).

So now we need to decide how to fix the code so that it doesn't error on the read. Antenna positions are required for the UVFits format, and in that case we spoof them with all zeros. Is that the mechanism we should use for Miriad files, or should they be left out in which case we should test to see if they exist when we read them in? @dannyjacobs @jpober do you have opinions?

**Comment by bhazelton** [comment]
Ian is also planning to fix FHD so it will capture and save this information, so this may go away on a timescale of a month or so, but I think we should fix it more generally before then.

**Comment by aelanman** [comment]
I have fixed it to not error on the read (branch "Default_antpos"), but it does issue a warning. That branch also has a modification of telescopes.py to read in default antenna positions from npz files.

**Comment by bhazelton** [comment]
Ok. We should fix this issue on master, though. I suspect the later change (reading in positions from npz files) can go away once this is fixed in FHD.

**Comment by aelanman** [comment]
I think I've got it resolved in the latest commit to Default_antpos. I've commented out anything to do with default antenna positions and removed the directory I'd made to hold the npz files.

**Comment by dannyjacobs** [comment]
so now we'll get the same behavior as uvfits?

**Comment by aelanman** [comment]
Should I submit a pull request?

**Comment by bhazelton** [comment]
@aelanman No. I don't think you've fixed it the way we ultimately want to fix it in the repo. And in general before you'd submit a pull request you'd need to rebase on master and make sure the tests are passing, which they don't seem to be on your branch (at least they're erroring on Travis).

@dannyjacobs I see two options. One is to handle it like uvfits where we spoof the missing information. The other is just to fix the logic to test for whether it's present or not in the file before demanding that it be there. The reason we went the way we did for uvfits is that antenna positions are required to have a valid uvfits file. What I was trying to ask you and @jpober is whether they are considered to be required for a Miriad file (and so we should have it behave the same way as uvfits) or not (in which case we should test for whether they're present or not).
